Workbooks/categoryResources.json

```diff
@@ -0,0 +1,13 @@
+{
+    "categories": [{
+        "key": "YourUniqueCategoryKey",
```

Comment:
instead of placeholder stuff, should we use this to create a new category like "Samples" or something, and put some sample workbooks into that?

there's another story somewhere to allow feature flags (or extra ui) to include extra categories/galleries.  this might be a good place to start, with samples?

Reply:
you marked this resolved but i don't see any comment or change?   why does this *real* categoryResources.json have placeholder example content?  shouldn't this have *real* content?  its ok if the docs have placeholder values like "YourUniqueCategoryKey" but why does this real file?

Reply:
We don't have any virtual category yet until Scott's team adds one. Once they add one, I will remove it.